### ISSUE TITLE
Should the BN layer be frozen?

### ISSUE THREAD

**Issue by taijizhao** [qqwweee/keras-yolo3#278]
Hi, all. I have a question and want to get some hint.
I see for many object detectors (such as MaskRCNN and RetinaNet), due to the small batch size,  the batch normalization layers are frozen. And this paper  
Wu Y, He K. Group normalization[J]. arXiv preprint arXiv:1803.08494, 2018.
reported that when training RetinaNet the BNs have to be frozen, otherwise the performance will be drastically decrease. 
I would suppose this also applies to Yolo v3. However, the BNs are not frozen and everything works well. 
I also tried to freeze all BN layers by 

for i in range(len(model.layers)):
    if str.startswith(model.layers[i].name, 'batch_normalization'):
        model.layers[i].trainable = False

But while training loss went down as normal but the validation loss does not decrease at all. 
What might be the problem and why yolo can train BN with only 2 images per batch (in my experiment)?
